packages/core/core/src/factories/useCartFactory.ts

```diff
@@ -38,21 +38,15 @@ interface UseCartFactory<CART, CART_ITEM, PRODUCT, COUPON> {
 export const useCartFactory = <CART, CART_ITEM, PRODUCT, COUPON>(
   factoryParams: UseCartFactoryParams<CART, CART_ITEM, PRODUCT, COUPON>
 ): UseCartFactory<CART, CART_ITEM, PRODUCT, COUPON> => {
-  const appliedCoupon: Ref<COUPON | null> = ref(null);
-  const loading: Ref<boolean> = ref<boolean>(false);
-  const cart: Ref<CART> = ref(null);
-  let isInitialized = false;
+  const appliedCoupon: Ref<COUPON | null> = vsfRef(null);
+  const loading: Ref<boolean> = vsfRef<boolean>(false);
+  const cart: Ref<CART> = vsfRef(null);
 
   const setCart = (newCart: CART) => {
     cart.value = newCart;
   };
 
   const useCart = (): UseCart<CART, CART_ITEM, PRODUCT, COUPON> => {
-    const { initialState, saveToInitialState } = useSSR('vsf-cart');
```

Comment:
So nuxt CAPI does not require any keys?

Reply:
About key from Nuxt CAPI docs:
```js
If you have added @nuxtjs/composition-api to your buildModules, this will be done automagically by an injected Babel plugin. 
```

I believe we do it like that so we do not need any keys - babel's plugin should take care of that